app/config/database.php

```diff
@@ -6,8 +6,11 @@
  * @author Tatsuya Tsuruoka <http://github.com/ttsuruoka>
  * @link https://github.com/dietcake/dietcake-showcase
  */
+require_once '/var/www/board_exercise/vendor/SimpleDBI/SimpleDBI.php';
+require_once '/var/www/board_exercise/vendor/SimpleDBI/SimpleDBIStatement.php';
```

Comment:
What if i installed your application in `/var/www/project/cyscorpions/board/`
I think you can just use the VENDOR_DIR constant?